optaplanner-website-root/content/learn/drl-to-constraint-streams-migration.adoc

```diff
@@ -0,0 +1,447 @@
+# Migrating OptaPlanner score DRL to Constraint Streams
+:jbake-description: Migrate deprecated Score DRL to Constraint Streams
```

Comment:
If they google search for "scoreDRL to constraint streams", this will be the paragraph under the link to convince them to click on it.

Soft suggestion: Let's make it it a bit more convincing:

```suggestion
:jbake-description: Upgrade your OptaPlanner scoreDRL constraints to the faster constraint streams constraints today.
```